<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterWndow</class>
 <widget class="QDialog" name="RegisterWndow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>422</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>注册</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>昵称:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="name_lineEdit">
       <property name="placeholderText">
        <string>大于3个字符</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>邮箱:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="email_lineEdit">
       <property name="placeholderText">
        <string>邮箱</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>密码:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="password_lineEdit">
       <property name="placeholderText">
        <string>大于6个字符</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>确定密码:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="password_2_lineEdit">
       <property name="placeholderText">
        <string>密码确认</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>验证码:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="emailCode_lineEdit">
       <property name="placeholderText">
        <string>邮箱收到验证码</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="send_email_pushButton">
       <property name="text">
        <string>发送验证码</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>邀请码</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="Invitation_code_lineEdit">
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>没有不用填</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QCheckBox" name="checkBox">
       <property name="text">
        <string>我已读并同意</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="link_tk_label">
       <property name="text">
        <string>本条款和条件</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="register_pushButton">
     <property name="text">
      <string>注册</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="to_longWinpushButton">
     <property name="text">
      <string>已经有账号?马上登录</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
